src/components/library_card_table/LibraryCardTable.jsx

```diff
@@ -133,15 +132,17 @@ const DeleteRecordButton = (props) => {
     setCurrentCard(card);
     setShowConfirmation(true);
   };
-  return h(TableIconButton, {
-    keyProp: `show-delete-modal-${card.id}`,
-    dataTip: 'Delete Library Card',
-    isRendered: true,
-    onClick,
-    icon: Delete,
-    style: Object.assign({}, Styles.TABLE.TABLE_ICON_BUTTON),
-    hoverStyle: Object.assign({}, Styles.TABLE.TABLE_BUTTON_ICON_HOVER)
-  });
+  return (
+    <TableIconButton
+      keyProp={`show-delete-modal-${card.id}`}
+      dataTip='Delete Library Card'
```

Comment:
I'm pretty sure this isn't working correctly but it's not related to this PR - it's likely a regression from a previous PR. For `ReactToolTip` in `TableIconButton` to work, there needs to be a `data-for` attribute on the target element.